### DISCUSSION TITLE
is there a way to convert  saml assertion to token claims (without storing them as role/group or user attribute)

### DISCUSSION THREAD

**Discussion by kkcmadhu** [keycloak/keycloak#16623] [Q&A]
I have an saml2.0 identity provider, I have some assertions coming in , I want to map those assertions into custom claims in my keycloak jwt token,
how can i achieve this.
I dont want to really store the incoming assertion into a role/group or user attribute..
I just want to get those attributes from saml assertion, not store anywhere in keycloak, and mint the info in token.

Is this possible?

**Comment by kapalex** [comment]
Interested in this as well
